be/src/runtime/memory/column_allocator.h

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <glog/logging.h>
+
+#include <memory>
+
+#include "runtime/memory/mem_hook_allocator.h"
+
+namespace starrocks {
+
+extern MemHookAllocator kDefaultColumnAllocator;
+inline thread_local Allocator* tls_column_allocator = &kDefaultColumnAllocator;
+
+template <class T>
+class ColumnAllocator {
```

Comment:
Why add a new allocator here, can we use `MemHookAllocator` or `CountingAllocator` directly?